mix.exs

```diff
@@ -33,6 +33,11 @@ defmodule SituationRoom.MixProject do
 
   defp aliases do
     # Alises that auto setup the database when conducting tests
-    [test: ["ecto.create --quiet", "ecto.migrate", "test"]]
+    [
+      test: [
+        "ecto.create --quiet",
+        "ecto.migrate", "test"
```

Comment:
Was this a manual reformat or automated?  If manual, curious why we took it from 1 line to several, but didnt fully split out the array into individual lines?  Is there a reason line 39 contains 2 elements still instead of having 3 total lines, one for each element?